provider/spark.go

```diff
@@ -1896,7 +1850,8 @@ func (spark *SparkOfflineStore) GetTransformationTable(id ResourceID) (Transform
 		return nil, err
 	}
 	spark.Logger.Debugw("Retrieved transformation source", "id", id, "filePath", transformationPath.ToURI())
-	return &FileStorePrimaryTable{spark.Store, transformationPath, TableSchema{}, true, id}, nil
+	// TODO: populate the schema
```

Comment:
Left on purpose?

Reply:
Yeah, I'd like to eventually populate the schema here, but I'm going to need a different approach than the primary table registration path.